### DISCUSSION TITLE
Semver calculator no longer includes pre-release versions

### DISCUSSION THREAD

**Discussion by IanVS** [community/community#128423] [npm]
Dealing with pre-release versions is one of the most complex parts of semver, and the calculator has been a helpful, albeit incomplete, tool when determining version ranges which include pre-releases.  

I wanted to double-check a range in the calculator today and found that _no_ pre-release versions are shown at all.  For example, Vite currently has 451 versions:

<img width="982" alt="image" src="https://github.com/npm/feedback/assets/4616705/972426a4-0137-43ae-8cba-75dc0f417aff">

But the calculator only includes 236, none of which are pre-releases:

<img width="696" alt="image" src="https://github.com/npm/feedback/assets/4616705/8dde45ec-f046-441f-8162-26b9f5460951">

What happened to them?

**Comment by KotlinIsland** [comment]
- Related: https://github.com/npm/node-semver/issues/605
